engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/interop/syntax/HostValueToEnsoNode.java

```diff
@@ -56,15 +66,30 @@ long doChar(char i) {
     return i;
   }
 
+  @Specialization(guards = {"n != null", "iop.fitsInBigInteger(n)"})
+  Object doBigIntegerConversion(
+      TruffleObject n,
+      @Shared("iop") @CachedLibrary(limit = "3") InteropLibrary iop,
+      @Cached ToEnsoNumberNode to) {
+    try {
+      if (n instanceof WithWarnings) {
```

Comment:
:) Shouldn't that be just a separate specialization?

Reply:
We shouldn't be referring to `WithWarnings` in specialization. Fixed in [9439797](https://github.com/enso-org/enso/pull/7420/commits/94397972891b19420b23dd566f5e35d29c1f5947)